pkg/game/test/factories.go

```diff
@@ -21,3 +21,25 @@ func GetTestScoreReport() elements.ScoreReport {
 		ReturnedMeeples: map[uint8][]uint8{},
 	}
 }
+
+func GetTestCustomPlacedTile(tileTemplate tiles.Tile, playerID uint8) elements.PlacedTile {
+	var placedFeatures []elements.PlacedFeature
+
+	// convert features to placedFeature
+	for _, feature := range tileTemplate.Features {
+		placedFeatures = append(placedFeatures, elements.PlacedFeature{
+			Feature: feature,
+			Meeple: elements.Meeple{
+				MeepleType: elements.NoneMeeple,
+				PlayerID:   elements.ID(playerID)},
+		})
+	}
```

Comment:
I'd suggest removing playerID from this function and just setting the Meeple's PlayerID to NonePlayer - currently it assigns a player to *every* feature on this tile (albeit meepleless), and later on you set the playerID manually anyways

Reply:
removed